readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -79,7 +76,7 @@ context = {
     'html_theme': html_theme,
     'current_version': "{{ version.verbose_name }}",
     'version_slug': "{{ version.slug }}",
-    'MEDIA_URL': "{{ settings.MEDIA_URL }}",
```

Comment:
We should probably continue to pass this along, in case anyone else is depending on it.

Reply:
In case we keep passing it, may be a good idea to have a comment saying that it's not need and should be removed but it's kept for compatibility reasons.

BTW, how we "deprecate" and communicate something like this? It's too small to worth a blog post, I think.